src/components/application_manager/src/commands/mobile/on_system_request_notification.cc

```diff
@@ -143,6 +144,21 @@ void OnSystemRequestNotification::AddHeader(BinaryMessage& message) const {
   LOG4CXX_DEBUG(
       logger_, "Header added: " << std::string(message.begin(), message.end()));
 }
+
+std::string OnSystemRequestNotification::ParsePTString(std::string& pt_string) const{
+  std::string result;
+  int length = pt_string.length();  
+  result.reserve(length*2);
+  for(int i=0;i<length;++i){
+    if(pt_string[i]=='\"' || pt_string[i]=='\\'){
```

Comment:
@pestOO This does not replace those characters, it just adds a backslash before those two characters are copied into the parsed string. Will point out line where original character is copied after backslash is added.